android/sdl_android/src/main/java/com/smartdevicelink/managers/video/resolution/AspectRatio.java

```diff
@@ -0,0 +1,19 @@
+package com.smartdevicelink.managers.video.resolution;
+
+public class AspectRatio {
```

Comment:
When `VideoSrteamingRange` is updated to match the proposal this class should no longer be needed

Reply:
@RHenigan Thanks for your reply, linking this comment with the conversation above https://github.com/smartdevicelink/sdl_java_suite/pull/1434#discussion_r572740386

Reply:
@kostyaBoss Please refer to the conversation above [#1434 comment](https://github.com/smartdevicelink/sdl_java_suite/pull/1434#discussion_r573179192)